File: ThemeTest.m

// This is a one-line c++ style comment http://url
/* This is a multi-
 line comment comment */

/*!
Documentation Comment
@param var
*/

#pragma mark: - My Section

#import <Foundation/Foundation.h>

- (void)methodName;

@interface MyObject : NSObject

@implementation MyObject

NSString *string = @"string";

FourCharCode code = 'aapl';

int i = 123;

NSArray *array = @[value];

@interface MyObject : NSObject
@property (nonatomic, readonly) BOOL flag;
@end

@implementation MyObject
- (instancetype)init
{
    self = [super init];
    if (self) {
    }
    return self;
}
@end

MyObject *object = [MyObject new];

typedef NS_ENUM(NSUInteger, MyEnum) {
    MyEnumValueA,
    MyEnumValueB,
    MyEnumValueC,
};

File: ThemeTest.swift

import UIKit

// This is a one-line c++ style comment http://url
/* This is a multi-
 line comment comment */

/*!
 Documentation Comment
 @param var
*/

// MARK: - My Section

var myString : String = "hell"
myString += "o"
let implicitDouble : Double = 70.0

print("\(myString) world")

var array : Array<Any> = [myString,implicitDouble]

func functionName(_ i: Int) -> Void {
    print(i)
}

for i in 0...2 {
    functionName(i)
}

class Parent {
    var name : String
    init(_ name : String){
        
        self.name = name
        
    }
    
    func fullname() -> String? {
        return "\(name) Appleseed"
    }
}

class Child : Parent { }

let johnny = Child("Johnny")

if let fullname = johnny.fullname(){
    
    print(fullname)
    
}

enum Cases: Int {
    case A = 1
    case B, C
    func simpleDescription() -> String {
        switch self {
        case .A:
            return "alpha"
        case .B:
            return "beta"
        default:
            return String(self.rawValue)
        }
    }
}

let card = Cases.A

var TimeInterval : TimeInterval = NSTimeIntervalSince1970
